operators/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![warn(missing_docs)]
+#![allow(missing_docs)]
```

Comment:
Any reason for this change?

Reply:
Was seeing errors in anyhow crate due to this macro.